src/iuse.cpp

```diff
@@ -4733,12 +4733,14 @@ cata::optional<int> iuse::blood_draw( player *p, item *it, bool, const tripoint
     item blood( "blood", calendar::turn );
     bool drew_blood = false;
     bool acid_blood = false;
+    float blood_temp;  //kelvins
```

Comment:
```suggestion
    float blood_temp = 0.0f;  //kelvins
```

Reply:
Why that default value?

The negative illegal indicator one mentioned in the referenced issue report would make some kind of sense, as would one that had some somewhat realistic "average reasonable" value (such as anywhere in the range 0 to 25 degrees Celsius). A default value of 0 degrees Kelvin is always going to be wrong, with the only advantage being that it probably doesn't cause any code to crash to indicate the result is nonsensical (unless you try to divide by temperature, but I can't currently see why you'd try to do that).

Reply:
> Why that default value?

In order to silence the compile warning.